src/components/onboarding.tsx

```diff
@@ -94,7 +94,10 @@ export default function Onboarding() {
 	);
 
 	return (
-		<div className="flex flex-row flex-grow">
+		<div
+			className="flex flex-row flex-grow"
+			data-testid="onboarding"
+		>
```

Comment:
We could actually remote this, given the current code doesn't use it because I couldn't get it to work.